tests/routes/webapp/get_beneficiary_profile_test.py

```diff
@@ -19,8 +19,19 @@ class Returns200:
         @pytest.mark.usefixtures("db_session")
         def when_user_is_logged_in_and_has_no_deposit(self, app):
             # Given
-            user = create_user(departement_code="93", email="toto@btmx.fr", public_name="Toto")
-            repository.save(user)
+            user = UserFactory(
+                civility="M.",
+                departementCode="93",
+                email="toto@btmx.fr",
+                firstName="Jean",
+                lastName="Smisse",
+                dateOfBirth=datetime(2000, 1, 1),
+                phoneNumber="0612345678",
+                postalCode="93020",
+                publicName="Toto",
+                isEmailValidated=True,
+            )
```

Comment:
Puisqu'on ne teste pas ces champs dans la sortie, on pourrait simplifier la définition de l'utilisateur :

```suggestion
            user = UserFactory(email="toto@btmx.fr")
```

idem dans les autres tests plus bas.